netket/graph/space_group.py

```diff
@@ -310,7 +310,6 @@ def one_arm_irreps(self, *k: Array) -> Array:
         """
         # Convert k to reciprocal lattice vectors
         k = _ensure_iterable(k)
-        k_rec = self.lattice.to_reciprocal_lattice(k) * (2 * pi / self.lattice.extent)
```

Comment:
This was removed because `k_rec` was unused, right? @attila-i-szabo Is the code here correct after removing `k_rec`? Just to make sure we don't overlook a bug.

Reply:
yes it was.

Reply:
doesn't seem to be used indeed.